.github/workflows/ci.yml

```
@@ -0,0 +1,45 @@
+name: Build Bluejay Firmware.
+
+on:
+  push:
+    branches:
+    - main
+    tags:
+    - '*'
+  pull_request:
+    branches:
+    - main
+
```

```suggestion
  workflow_dispatch:
    branches:
    - main
    
```
So to make it can be triggered manually (for official releases like RCx and final release etc).

---

Hi, For official releases, the idea is to do a git tag vX.Y.Z and git push, and that should generate the changelog and create releases and upload the artifacts.

Just waiting for @stylesuxx 's input on if we should compress the hex files into a zip before we upload, because the releases page is slowing down a lot when it tries to load 1000+ files